### PR TITLE
Add record programs, keeps old consoles to avoid converting main maps.

### LINKED ISSUE #6042: Records are now so secure you cannot print them anymore.
What the title says. The update to records makes it impossible to print them, due to the absence of a print button.

### PR DESCRIPTION
**Pull request by Karolis2011**
- [x] Add record printing (fixes #6042)

### CODE REVIEW

**Comment by MarinaGryphon** [comment]
Are records editable via these?

**Comment by Karolis2011** [comment]
Like nothing really changed... Thoes programs so far work exactly like consoles. Even login button is there. Record refactor was being done with this being a thing. 

Honestly my dream was to have records on modular computers, but things needed to have them were just too long like three major projects were to just accomplish this dream 1 - VueUi to build eveneryting on top of it, 2 - Records being refactored to be better and new UI and 3 - Modular computers supporting VueUi. And this is probably last installment in this saga.

**Comment by Karolis2011** [comment]
paveikslas

**Comment by Karolis2011** [comment]
This PR needs sprites and map fixes. 
for now 
!review

**Comment by alsoandanswer** [comment]
"Phisical status"

File: code/modules/modular_computers/file_system/programs/generic/records.dm [hunk 1]
```diff
@@ -1,6 +1,17 @@
-/obj/machinery/computer/records
-	name = "records console"
-	desc = "Used to view, edit and maintain records."
+/datum/computer_file/program/records
+	filename = "record"
+	filedesc = "Recrods"
```
Comment (alsoandanswer) [inline code/modules/modular_computers/file_system/programs/generic/records.dm]:
```suggestion
	filedesc = "Records"
```

**Comment by fernerr** [comment]
DNM until the mapping stuff is merged into this branch.

**Comment by fernerr** [comment]
Message me when the other conflicts are fixed and I'll fix the map ones.

**Comment by Karolis2011** [comment]
Real life got too busy. I don't have enough time for this till I be able to just manage fix merge conflicts fast enough to be merged.

**Comment by Karolis2011** [comment]
!review

**Pull request merged**